package.json

```diff
@@ -18,7 +18,7 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject",
     "cypress": "cypress open",
-    "tscompile": "tsc -p ./tsconfig.json"
+    "tscompile": "tsc -p cypress/tsconfig.json --noEmit"
```

Comment:
The Typescript compiler has to be isolated to the correct project root. The `cypress/tsconfig.json` is what defines what namespace types to include with the project.